src/Cake.Common/Tools/ReportGenerator/ReportGeneratorSettings.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.ReportGenerator
+{
+    /// <summary>
+    /// Contains settings used by <see cref="ReportGeneratorRunner"/>.
+    /// </summary>
+    [CakeAliasCategory("ReportGenerator")]
+    public sealed class ReportGeneratorSettings
```

Comment:
@gep13 @yvschmid @devlead All tools should have an optional tool path. Not everyone has one, but they should :)¨

About the `ReportGeneratorSettings`, there is no need for a `CakeAliasCategory` attribute here since this is not an alias.